Packs/BitcoinAbuse/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "BitcoinAbuse",
+    "description": "BitcoinAbuse Integration Pack",
+    "serverMinVersion": "5.5.0",
```

Comment:
```suggestion
```

Reply:
can I ask why should it be deleted? as this integration is feed I understand it's only relevant for 5.5.0 or above

Reply:
leave it